assets/scripts/utils/visibility.js

```diff
@@ -29,15 +29,15 @@
  * - {@link https://www.w3.org/TR/page-visibility/ W3 Specification}
  * - {@link https://developer.mozilla.org/en-US/docs/Web/API/Page_Visibility_API MDN Web Docs}
  */
-export default new class PageVisibility {
+export default new (class PageVisibility {
```

Comment:
Why is this class wrapped in a IIFE?

Reply:
Prettier might wrap a singleton class export in an IIFE to avoid multiple instantiation of the class. By wrapping the singleton class in an IIFE, the instance of the class is created only once and then cached for future references. This ensures that every time the module is imported, the same instance of the class is returned